deploy-as-code/helm/environments/pspcl-ifix-uat.yaml

```diff
@@ -59,7 +59,7 @@ cluster-configs:
         fiscal-event-service: "http://fiscal-event-service.ifix:8080/"
         ifix-zuul: "http://ifix-zuul.ifix:8080/"
         ifix-department-entity-service: "http://ifix-department-entity-service.ifix:8080/"
-        egov-mdms-service: "http://egov-mdms-service.mgramseva:8080/"
+        egov-mdms-service: "https://mgramseva-uat.psegs.in/"
```

Comment:
Service host names should be read from config maps. 
Add the correct mdms host name in the config map, and change the helm chart to read the value from it.

Reply:
added in egov-config mdms host

Reply:
Why isn't "http://egov-mdms-service.mgramseva:8080/" this working?